src/elements/Label/Label.js

```diff
@@ -35,31 +34,34 @@ function Label(props) {
     useKeyOnly(circular, 'circular'),
     useKeyOnly(floating, 'floating'),
     useKeyOnly(horizontal, 'horizontal'),
+    useKeyOnly(image, 'image'),
     useKeyOnly(empty, 'empty'),
     useKeyOnly(tag, 'tag'),
     useValueAndKey(attached, 'attached'),
     useKeyOrValueAndKey(corner, 'corner'),
     useKeyOrValueAndKey(pointing, 'pointing'),
+    /* Don't change order of `above` and `below`, they must be after `pointing` */
+    useKeyOnly(above, 'above'),
+    useKeyOnly(below, 'below'),
```

Comment:
> I think these should be _meta.props.pointing values.

I think, too, but `useKeyOrValueAndKey(pointing, 'pointing')` will produce `above pointing` while it must be [`pointing above`](https://github.com/Semantic-Org/UI-Label/blob/master/label.css#L1157).